src/arb/arb.jl

```diff
@@ -2166,46 +2166,21 @@ end
 ################################################################################
 
 @doc Markdown.doc"""
-    rand(rng::AbstractRNG, r::ArbField; randtype::Symbol=:null)
+    rand(r::ArbField; randtype::Symbol=:null)
 
 Return a random element in the given Arb field. The values are distributed
 non-uniformly in order to exercise corner cases.
-"""
-function rand(rng::AbstractRNG, r::ArbField; randtype::Symbol=:null)
-  state = rand_ctx()
-  ccall((:flint_randseed, libarb), Nothing, (Ptr{Cvoid}, UInt, UInt),
```

Comment:
I know I'm comenting on a deleted function, but for info: passing an `rng` here would have been confusing, as it was used only to initialize the implicit flint RNG. It's best to decouple reseeding an RNG from actually producing random values. For seeding, we now have the `Nemo.randseed!` function, which re-seeds the two Flint global RNGs.

Reply:
Pinging @tthsqe12, since he asked about reseeding the flint rng a few days ago.

Reply:
Thanks. Nemo.randseed! is exactly what I was looking for